src/pages/Home/component/Education/index.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Education } from "../../../../data/data";
+
+import {
+  StyledHomeEducation,
+  StyledHomeEducationArticle,
+  StyledHomeEducationSection,
+} from "./styles";
+
+const index = () => {
+  return (
+    <StyledHomeEducation>
+      <h2>My Education & Experience</h2>
+      <StyledHomeEducationSection>
+        <StyledHomeEducationArticle>
+          <header> 2000 </header>
+          <span>
+            <h5>{Education.first_part.title}</h5>
+            <p>{Education.first_part.caption}</p>
+          </span>
+        </StyledHomeEducationArticle>
+        <StyledHomeEducationArticle>
+          <header> 2017 - 2019 </header>
```

Comment:
what about 2019-2020? where you were working, it is better to add months as well. add other company names there like `Javidan company` which you were working as a frontend developer doing template design